### PR TITLE
Move to ubi-micro

### PR DESCRIPTION
**Pull request by qbarrand**
Move the manager binary to /usr/local/bin ([MGMT-12589](https://issues.redhat.com//browse/MGMT-12589)).
Use a TLS bundle provided by OpenShift ([MGMT-12594](https://issues.redhat.com//browse/MGMT-12594)).
Upgrade to the latest operator-sdk.

/cc @fabiendupont @cdvultur @mresvanis

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,10 +25,9 @@ COPY .git .git
...
-COPY --from=builder /workspace/manager .
-RUN useradd -r kmm
-USER kmm:kmm
+FROM registry.redhat.io/ubi8/ubi-micro:8.7
 
-ENTRYPOINT ["/manager"]
+COPY --from=builder /workspace/manager /usr/local/bin
+USER 65534:65534
+
+ENTRYPOINT ["manager"]
```
Comment (fabiendupont) [inline Dockerfile:33]: I usually use an absolute path, just in case another program with the same name exists a higher priority path, like `/usr/bin`.

Comment (qbarrand) [reply Dockerfile:33]: I used `manager` to remain compatible with possible future changes, but your point makes sense. PTAL.

File: bundle/manifests/kmm-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -239,8 +247,8 @@ spec:
                 - --metrics-bind-address=127.0.0.1:8080
                 - --leader-elect
                 command:
-                - /manager
-                image: quay.io/edge-infrastructure/kernel-module-management:latest
+                - /usr/local/bin/manager
+                image: quay.io/quba/kmmo:latest
```
Comment (cdvultur) [inline bundle/manifests/kmm-operator.clusterserviceversion.yaml:251]: Why do you point to your quay account ? Maybe is from testing ?

Comment (qbarrand) [reply bundle/manifests/kmm-operator.clusterserviceversion.yaml]: Should be fixed - apologies

**Review by mresvanis — APPROVED** [review]

**Pull request merged**